tests/datatypes/test_str.py

```diff
@@ -778,6 +778,43 @@ def test_splitlines(self):
         print(s1.splitlines(True))
         """)
 
+    def test_zfill(self):
+        self.assertCodeExecution("""
+            s = '42'
+            print(s.zfill(5))
+        """)
+
+    @expectedFailure
+    def test_zfill_missing_arg(self):
+        self.assertCodeExecution("""
+            s = '42'
+            print(s.zfill())
```

Comment:
This should not be an `@expectedFailure` (as this would allow any kind of failure), but instead it should just catch the right exception and print the exception -- the output should be the same as CPython.
Look at the tests for `rpartition` to see examples of this.